code/game/objects/items/weapons/storage/boxes.dm

```diff
@@ -593,3 +594,67 @@
 	new /obj/item/ammo_casing/shotgun/beanbag(src)
 	new /obj/item/ammo_casing/shotgun/beanbag(src)
 	new /obj/item/ammo_casing/shotgun/beanbag(src)
+
+/obj/item/weapon/storage/box/papersack
+	name = "paper sack"
+	desc = "A sack neatly crafted out of paper."
+	icon_state = "paperbag"
+	item_state = "paperbag"
+	burn_state = FLAMMABLE
+	foldable = null
+	var/design = 0 /* 0 = No design, 1 = Nanotrasen Standard, 2 = Syndisnacks, 3 = Heart, 4 = Smiley Face */

```

Comment:
use defines instead of weird numbers you have to comment on

#define NANOTRASEN "Nanotrasen Standard" etc